modules/ossm-mixer-policy-1x.adoc

```diff
@@ -26,7 +27,7 @@ $ oc get cm -n istio-system istio -o jsonpath='{.data.mesh}' | grep disablePolic
 +
 [source,terminal]
 ----
-$ oc edit cm -n istio-system istio
+$ oc edit cm -n istio-<smcp-name> istio
```

Comment:
This instance is still remaining, while it is fixed in `module/ossm-mixer-policy.adoc`.

Reply:
Cheers! Done...